components/calendar/benches/date.rs

```diff
@@ -0,0 +1,165 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+mod fixtures;
+
+use crate::fixtures::structs::DateFixture;
+use criterion::{
+    black_box, criterion_group, criterion_main, measurement::WallTime, BenchmarkGroup, Criterion,
+};
+use icu_calendar::{
+    iso::IsoDay, iso::IsoMonth, iso::IsoYear, AsCalendar, Calendar, Date, DateDuration,
+};
+
+fn bench_dates<A: AsCalendar>(dates: Vec<&mut Date<A>>) {
```

Comment:
As suggested below: this function should accept a `&mut [Date<A>]` rather than `Vec<&mut Date<A>>`

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=b4be5acb10b8715fd8857ac065a97fea

Reply:
Done